pkg/serveswagger/swagger_test.go

```diff
@@ -0,0 +1,27 @@
+package serveswagger
+
+import (
+	"os/exec"
+	"regexp"
+	"testing"
+)
+
+func TestCheckTimestamps(t *testing.T) {
+	cmd := exec.Command("git", "diff", "../../pkg/serverswagger")
```

Comment:
Before running `git diff`, this needs to re-generate the code with `go generate ./cmd/go-swagger/restapi`. Otherwise, it's not actually testing anything.

Also, `serverswagger` is a typo - it should be `serveswagger`.

Reply:
Also, please fix git so that you can test locally.